src/main/java/seedu/address/logic/commands/DeleteAppointmentCommand.java

```diff
@@ -52,7 +55,20 @@ public CommandResult execute(Model model) throws CommandException {
         // Delete appointment from model and patient
         Appointment appointmentToDelete = lastShownAppointmentList.get(zeroBasedAppointmentIndex);
         Person patient = appointmentToDelete.getPerson();
+        Doctor targetDoctor = null;
+        Name doctorName = new Name(appointmentToDelete.getName());
+        for (Doctor doctor : doctorList) {
```

Comment:
Is there a List function to find doctor in list by name?

Reply:
Agree with this. Perhaps can use .get( ) method of List library

Reply:
Agree with this. Perhaps can use .get( ) method of List library